src/storage/documentService/documentService.js

```diff
@@ -10,6 +10,10 @@ const { put, get, remove } = require("../s3");
 
 const DEFAULT_TTL_IN_MICROSECONDS = 30 * 24 * 60 * 60 * 1000; // 30 Days
 const MAX_TTL_IN_MICROSECONDS = 90 * 24 * 60 * 60 * 1000; // 90 Days
+const CORS_ERROR_HEADER = {
```

Comment:
what are the considerations you made when locating this piece of code in this file?

Reply:
this seems like the file with the code of how the endpoints interact with the db. thought it might be ok to include the cors_error_header inside here as most error might be due to 'not able to upload' to db, for instance connection error. and also conveniently is the only file it imports written by us. but it does seem weird to put it inside here.

Reply:
what would you suggest as less weird options?

Reply:
i think can create another file and put the const variables (DEFAULT_TTL..., MAX_TTL..., CORS_ERROR...) inside. Then can put comment indicating which file(s) uses which variable?